runtime/common/src/transfer_filter.rs

```diff
@@ -175,23 +172,32 @@ impl<T: TransferAllowance<AccountId, CurrencyId = FilterCurrency, Location = Loc
 	}
 }
 
-#[derive(Clone, Copy, PartialOrd, Ord, PartialEq, Debug, Eq, Encode, Decode, TypeInfo)]
-pub struct PreBalanceTransferExtension<T: Sync + Send + sp_std::fmt::Debug + TypeInfo>(
-	sp_std::marker::PhantomData<T>,
-);
+#[derive(Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Encode, Decode, TypeInfo)]
+#[scale_info(skip_type_params(T))]
+pub struct PreBalanceTransferExtension<T: frame_system::Config>(sp_std::marker::PhantomData<T>);
+
+impl<T: frame_system::Config> PreBalanceTransferExtension<T> {
+	pub fn new() -> Self {
+		PreBalanceTransferExtension(sp_std::marker::PhantomData::default())
+	}
+}
+
+impl<T: frame_system::Config> sp_std::fmt::Debug for PreBalanceTransferExtension<T> {
+	fn fmt(&self, f: &mut Formatter<'_>) -> sp_std::fmt::Result {
+		f.debug_struct("PreBalanceTransferExtension").finish()
+	}
+}
```

Comment:
Maybe you were looking for the derive `RuntimeDebugNoBound` (?)

Reply:
Nice.